src/components/setting/setting.js

```diff
@@ -1,151 +1,199 @@
+              <label className={`${styles.fieldGroup} ${styles.checkboxField}`}>
+                <CheckBox
+                  name={'autoLog'}
+                  className={`${styles.checkbox} autoLog`}
+                  checked={settings.autoLog}
+                  onChange={this.toggleAutoLog}
+                  />
+                  <div>
+                  <span className={styles.labelName}>{t('Auto Logout')}</span>
+                  <p>{t('Log out automatically after a specified amount of time.')}</p>
+                  </div>
+              </label>
+              <div className={`${styles.fieldGroup} ${styles.checkboxField} second-passphrase`}>
+                <div className={isHwWalletClass}>
+                  <span className={styles.labelName}>{t('Second Passphrase')}</span>
+                  <p>
+                    {t('Every time you make a transaction you’ll need to enter your second passphrase in order to confirm it.')}
+                  </p>
+                  <p className={styles.highlight}>{t('Once activated can’t be turned off.')}</p>
+                  {!hasSecondPassphrase ?
+                    <SignInTooltipWrapper>
+                      <Link
+                        className={`register-second-passphrase ${styles.link}`}
+                        to={`${routes.secondPassphrase.path}`}>
+                        {t('Activate (5 LSK Fee)')}
+                      </Link>
+                    </SignInTooltipWrapper>
+                  : (
+                    <span className={styles.activeLabel}>
+                      {t('Second Passphrase is active for this account.')}
+                    </span>
+                  )}
+                </div>
+              </div>
+            </section>
+            <section>
+              <h1>{t('Advanced')}</h1>
+              <label className={`${styles.fieldGroup} ${styles.checkboxField}`}>
+                <CheckBox
+                  name={'showNetwork'}
+                  className={`${styles.checkbox} showNetwork`}
+                  checked={settings.showNetwork}
+                  onChange={this.handleCheckboxChange}
+                />
+                <div>
+                  <span className={styles.labelName}>{t('Network switcher')}</span>
+                  <p>{t('Enable a network switcher that lets you select testnet or custom node when logging in.')}</p>
+                </div>
+              </label>
+              <label className={`${styles.fieldGroup} ${styles.checkboxField}`}>
+                <CheckBox
+                  name={'BTC'}
+                  className={`${styles.checkbox} enableBTC`}
+                  checked={settings.token.list.BTC}
+                  onChange={this.handleTokenToggle}
+                />
+                <div>
+                  <span className={styles.labelName}>{t('BTC token')}</span>
+                  <p>{t('By enabling it, you will be able to manage your BTC tokens inside the application.')}</p>
+                </div>
```

Comment:
IMO should be displayed only if `localStorage.getItem('btc');`

Reply:
Fixed